public/scripts/open-project.js

```diff
@@ -0,0 +1,115 @@
+(function() {
+  require(["jquery"], function($) {
+    var projects = document.querySelectorAll("tr.bramble-user-project");
+
+    function generateUrl(path, date) {
+      var url = window.location;
+      var qs = url.search;
+      if(qs.length < 1) {
+        qs = "?";
+      }
+      qs += "now=" + encodeURIComponent(date);
+
+      return url.protocol + "//" + url.host + "/newProject/" + encodeURIComponent(path) + qs;
+    }
+
+    $("#project-submit").on("click", function(e) {
+      e.preventDefault();
+
+      var projectName = document.getElementById("project-name");
```

Comment:
This is actually just getting whatever the user types in the input box for the name of a new project